examples/observability_archive/main.tf

```diff
@@ -91,4 +135,22 @@ module "observability_instance_creation" {
   at_cos_bucket_name                = local.bucket_name
   at_cos_instance_id                = module.cos.cos_instance_id
   at_cos_bucket_endpoint            = module.cos.s3_endpoint_private
+  cloud_logs_retention_period       = 14
+  cloud_logs_region                 = "eu-es"
```

Comment:
https://cloud.ibm.com/docs/log-analysis?topic=log-analysis-deprecation#:~:text=IBM%20Cloud%20Logs%20will%20become,the%20IBM%20Cloud%20Logs%20documentation. 

Should the code allow all regions and let the service fail back through the provider, then through the module if it is not supported yet? There are wider considerations, including variables.tf